generated/nidaqmx/system/device.py

```diff
@@ -589,6 +636,68 @@ def bus_type(self):
         val = self._interpreter.get_device_attribute_int32(self._name, 0x2326)
         return BusType(val)
 
+    @property
+    def cal_acc_connection_count(self):
+        """
+        int: Specifies the number of times a particular connection that
+            results in tangible wear and tear of onboard components has
+            been made on the accessory. This connection count is useful
+            for tracking accessory life and usage.
+        """
+
+        val = self._interpreter.get_cal_info_attribute_uint32(0x2feb)
+        return val
+
+    @cal_acc_connection_count.setter
+    def cal_acc_connection_count(self, val):
+        self._interpreter.set_cal_info_attribute_uint32(0x2feb, val)
+
+    @property
+    def cal_dev_temp(self):
+        """
+        float: Indicates in degrees Celsius the current temperature of
+            the device.
+        """
+
+        val = self._interpreter.get_cal_info_attribute_double(0x223b)
```

Comment:
@bkeryan @zhindes 
I find this `get_cal_info_attribute_double(0x223b)` is missing `self._name` argument when I write test case, same case on other calibration info attributes. After comparing existing attributes, `handle_parameters` are not available on the calibration info attributes in the [metadate - attributes.py](https://github.com/ni/nidaqmx-python/blob/master/src/codegen/metadata/attributes.py).

```
'handle_parameters': {
                'deviceHandle': {
                    'accessor': 'self._name',
                    'ctypes_data_type': 'ctypes.c_char_p',
                    'cvi_name': 'deviceName'
                }
            },
```

[CalDevTemp attribute](https://github.com/ni/nidaqmx-python/blob/c5fddf5c9dd6be9b0d02bcd96338963176515cf3/src/codegen/metadata/attributes.py#L195)

Is [metadate - attributes.py](https://github.com/ni/nidaqmx-python/blob/master/src/codegen/metadata/attributes.py) handwritten or codegen in another repo?

Reply:
its codegen'd from scrapigen, I'll point you to it offline

Reply:
This is resolved, right? After you close this, you can merge the PR.

Reply:
Yes, it is resolved. The changes have been reflected `cal_dev_temp`, `ext_cal_last_temp` and `self_cal_last_temp` in the device.py.

![image](https://github.com/ni/nidaqmx-python/assets/27721199/e5e0dc60-c147-448f-8ea9-f52d4dc20f9f)